vector/src/main/res/values/strings.xml

```diff
@@ -25,8 +25,8 @@
     <string name="notice_direct_room_leave_by_you">You left the room</string>
     <string name="notice_room_reject">%1$s rejected the invitation</string>
     <string name="notice_room_reject_by_you">You rejected the invitation</string>
-    <string name="notice_room_kick">%1$s kicked %2$s</string>
-    <string name="notice_room_kick_by_you">You kicked %1$s</string>
+    <string name="notice_room_kick">%1$s removed %2$s</string>
```

Comment:
Can you add a TODO in XML comment above so that I will not forget to rename during the next weblate sync. Thanks. (only one comment will be enough)